guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -109,6 +109,37 @@ baseurl=file:///media/sat6/Satellite
 baseurl=file:///media/sat6/Maintenance/
 ----
 
+ifdef::katello,satellite,orcharhino[]
+. If you use {advisorengine} in your environment, update to the latest images.
+
+.. You can use the {Project} installation ISO image to update the {advisorengine} container content.
+... Download and mount the {Project} ISO image and set up the local repositories.
+For more information, see {InstallingServerDisconnectedDocURL}installing-from-the-offline-repositories[Installing the {Project} packages from the offline repositories] in _{InstallingServerDisconnectedDocTitle}_.
```

Comment:
I think this should link to "[2.2. Configuring the base operating system with offline repositories](https://docs.redhat.com/en/documentation/red_hat_satellite/6.17/html-single/installing_satellite_server_in_a_disconnected_network_environment/index#configuring-the-base-operating-system-with-offline-repositories_satellite)" as that's where the ISO is mounted. Repository setup is not required for Advisor tho.

On the other hand, this is exactly what's described above in "Create directories to serve as a mount point, mount the ISO images" -- so we might just skip that here?

Reply:
Removed the step completely.